### ISSUE TITLE
Would it be useful to have Chapel docs at DevDocs?

### ISSUE THREAD

**Issue by bradcray** [chapel-lang/chapel#11136]
This issue asks the question whether it would be useful to have Chapel documentation at DevDocs and what the maintenance burden of doing so would be?

https://devdocs.io

**Comment by aditmehta9** [comment]
Yes, it would really useful in fact more than 100,000 developers use DevDocs. Developers will notice and contribute more in chapel. I want to contribute to this but I don't know how to start please guide me through this @bradcray.

**Comment by bradcray** [comment]
I don't really know enough about DevDocs to be able to guide you in any particular direction.  I filed this issue after someone asked about it as a placeholder, but would need to invest more time to determine what we should do.  I'd be open to you proposing a path.  My main concern is to avoid adding (very much) maintenance burden to our plate.  I.e., the ideal would be to have some way of autoreflecting the docs at https://chapel-lang.org/docs/ on DevDocs so that they'd automatically be in sync without any effort on our part.  Next best would be to have a single button we'd need to push (or script we'd need to run) to cause that to happen.  Anything more than that would probably start to feel like a maintenance burden after awhile unless we got positive feedback from users that they were there.

**Comment by aditmehta9** [comment]
Thanks, Sir, I'll add Chapel documentation in DevDocs in GSoC period

**Comment by immadisairaj** [comment]
@bradcray I just now went through the DevDocs and found out the criteria to add the documentation - [link](https://trello.com/c/PVnfdeaN/26-suggest-new-docs-here). One of it says that the `Project must have >5k stars on GitHub or equivalent` which is not the case with Chapel (1.1k stars). Also there is one more thing, I didn't understand for sure, but from what they have in their readme is that we might have to make a new PR for any documentation changese - [link](https://github.com/freeCodeCamp/devdocs/blob/master/.github/CONTRIBUTING.md#updating-existing-documentations). I guess it would be a bit maintenance burden if that is the case.

**Comment by bradcray** [comment]
Thanks for looking into it @immadisairaj!  Only ~3900 stars to go, I guess.  :)

**Comment by Aniket21mathur** [comment]
@immadisairaj I agree with you it would be a maintenance burden but at the same time it is a really good platform to support documentations and it might increase the no. of users hitting the chapel docs and will for sure make it easier for the current users. Agreeing with @bradcray let's keep it something to be discussed further in the future :smile:
